README.md

```diff
@@ -87,7 +87,23 @@ Finally you can boot the rails app:
 ```bash
 rails s
 ```
+#### Docker Compose
 
+If you're familiar with [Docker](https://docs.docker.com/engine/) and [Docker Compose](https://docs.docker.com/compose/), the included `docker-compose.yml` configuration allows you to spin up the application locally.
```

Comment:
Probably best to link to: https://docs.docker.com/installation/

Docker is actually of more value if you're not familiar with it. My normal instructions here are:

Step 1: [install Docker for Mac/Windows.](https://docs.docker.com/installation/)
Step 2: run `docker-compose up`

Boom.